README.md

```diff
@@ -30,30 +30,58 @@ testTools(gulp, {
 
 ```
 
-Running `gulp help` will now show all the tasks and a description (if provided) for each of the tasks.
+Running `gulp help` will show all the tasks and a description (if provided) for each of the tasks.
 
 
 ### Options
+ - es6: Specifies that your code is written in ES6 (+ JSX). You must also name your gulpfile `gulpfile.babel.js`. Defaults to `false`.
```

Comment:
It just seems weird to me that we'd always apply linting rules for JSX when a project may just be regular ES6. Maybe there won't be any style/linting conflicts and it doesn't matter?